src/libtiled/mapreader.h

```diff
@@ -95,6 +97,9 @@ class TILEDSHARED_EXPORT MapReader
      */
     QString errorString() const;
 
+	int GetNumTilesetSourceFiles();
+	QString* GetTilesetSourceFiles();
```

Comment:
Well, I had noted that already in another comment. But actually all this code is simply not necessary, as I wrote in later comments. So it should just be removed rather than fixed.